core/src/main/java/hudson/tasks/_maven/MavenMojoNote.java

```diff
@@ -57,5 +58,6 @@ public String getDisplayName() {
         }
     }
 
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
```

Comment:
Same?

Reply:
I guess noone will try to modify the pattern with script console, so I'm setting it final.